### DISCUSSION TITLE
XY color plot based on Z

### DISCUSSION THREAD

**Discussion by montpy** [facontidavide/PlotJuggler#833] [General]
Hi this is my first interaction with the community.  I am enjoying using PJ.  Thanks for this great tool.
I am making a longitude, latitude plot of robot position by displaying topic data in real time.

Q1:  I would like to color this plot based on an accuracy metric.  For example, green when accuracy is good, red when accuracy is bad.  If I have an accuracy data member in my message, how can I achieve this in plotjuggler?

Q2:  I would like to specify the limits of my XY plot.  For example set the X axis to display the range [xmin xmax] and set the Y range to display the range [ymin ymax].  I can set the vertical limits in the Edit Curves pulldown menu, but not the X range.  Is there a way to do this?  

Many thanks for any advice.

**Comment by paulmteleo** [comment]
With reference to Q2:  It's a bit of a pain, but one can create a .csv file with a desired boundary of XY points. By adding that to the canvas, I can effectively set the [xmin,xmax] [ymin, ymax].  But its far from convenient.

**Comment by facontidavide** [comment]
not happening
